### ISSUE TITLE
[Bug] doesnt launch on linux

### ISSUE THREAD

**Issue by jiscaskalov** [Lyzev/Schizoid#152]
### Describe the Bug

client fails to launch but still shows in mod menu
os: arch linux

[crash report](https://mclo.gs/3ZxOkfL) (ignore the narrator error that is a mc issue)

### Steps to Reproduce

```markdown
install arch linux and launch client
```

### Expected Behavior

the client launches

### Actual Behavior

it doesnt launch

### Operating System

Linux

### Before Submitting a Bug Report

- [X] I have checked that this bug was not already reported.

- [X] I am able to reproduce this bug on the latest development build.

**Comment by Redstonecrafter0** [comment]
Transient dependencies are not shipped in the media player library. This is done to reduce the binary itself and provide the ability to override the transient dependency version.

It looks like the java dbus library is missing. Transient dependencies should be shipped in the final jar used by the end user.

TLDR: The issue is not the library but the build step not including transient dependencies.

**Comment by Lyzev** [comment]
This version should fix the bug: https://github.com/Lyzev/Schizoid/releases/tag/v0.1.0%2Bdb43cb1

If it doesn't, let me know and I reopen this issue.

**Comment by jiscaskalov** [comment]
> This version should fix the bug: https://github.com/Lyzev/Schizoid/releases/tag/v0.1.0%2Bdb43cb1
> 
> If it doesn't, let me know and I reopen this issue.

It doesn't seem to launch still, but a different error: [log](https://mclo.gs/ISdULu7)

**Comment by Lyzev** [comment]
Here we go again...

**Comment by Lyzev** [comment]
hopefully this version works now: https://github.com/Lyzev/Schizoid/releases/tag/v0.1.0%2B6374edc

**Comment by Lyzev** [comment]
still doesn't work...

**Comment by Lyzev** [comment]
This should fix the bug: https://github.com/Lyzev/Schizoid/releases/tag/v0.1.0-219db9f%2B1.21
I tested it on Linux Mint.
